packages/apps/docs/src/components/Layout/Blog/Blog.tsx

```diff
@@ -1,39 +1,46 @@
-import { Stack } from '@kadena/react-ui';
-
 import { Article, Content, TitleHeader } from '../components';
 import { Template } from '../components/Template';
 
+import { ArticleMetadataItem, ArticleTopMetadata } from './BlogStyles';
 import { PageGrid } from './styles';
 
 import { BottomPageSection } from '@/components/BottomPageSection';
 import { IPageProps } from '@/types/Layout';
-import { formatISODate } from '@/utils/dates';
+import { formatDistance } from 'date-fns';
 import React, { FC } from 'react';
 
 export const Blog: FC<IPageProps> = ({
   children,
   frontmatter,
   leftMenuTree,
 }) => {
+  const { readingTimeInMinutes, publishDate, author } = frontmatter;
+
   return (
     <PageGrid>
       <Template menuItems={leftMenuTree} hideSideMenu layout="landing">
         <TitleHeader
-          title={frontmatter.title}
-          subTitle={frontmatter.subTitle}
-          icon={frontmatter.icon}
+          title="BlogChain"
+          subTitle="The place where the blog meets the chain"
+          icon="BlogChain"
         />
 
         <Content id="maincontent">
```

Comment:
the content is not aligned in the center of the page.
check the text and compare it to the main header.

<img width="1366" alt="image" src="https://github.com/kadena-community/kadena.js/assets/4015521/685cd420-9df4-4756-9964-ef431d473b67">